### PR TITLE
Add docs link to skeleton daml.yaml.template

### PR DESCRIPTION
**Pull request by shaul-da**
### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

**Comment by bame-da** [comment]
I'm unopinionated on this, but if we do it, I'd put the comment at the top of the file. What I have seen done is to have a verbose config with all common options, but most of them commented out.

File: templates/skeleton/daml.yaml.template [hunk 1]
```diff
@@ -12,3 +12,5 @@ dependencies:
   - daml-script
 sandbox-options:
   - --wall-clock-time
+  
+# for more config file options, refer to https://docs.daml.com/tools/assistant.html#project-config-file-daml-yaml
```
Comment (hurryabit) [inline templates/skeleton/daml.yaml.template:16]: Let's spread it over two lines to make the lines not too long.
```suggestion
# for more config file options, refer to
# https://docs.daml.com/tools/assistant.html#project-config-file-daml-yaml
```

**Review by hurryabit — APPROVED** [review]
I like the idea very much and IIRC other languages do that too.

**Comment by hurryabit** [comment]
@shaul-da You also need a commit containing
```
CHANGELOG_BEGIN
CHANGELOG_END
```
in its commit message.

**Comment by shaul-da** [comment]
> I'm unopinionated on this, but if we do it, I'd put the comment at the top of the file. What I have seen done is to have a verbose config with all common options, but most of them commented out.

Yep, I agree. Didn't feel I'm qualified for that so I went for the poor man's version

**Pull request closed without merging**